mlflow/store/tracking/rest_store.py

```diff
@@ -541,6 +552,20 @@ def record_logged_model(self, run_id, mlflow_model):
         )
         self._call_endpoint(LogModel, req_body)
 
+    def get_logged_model(self, model_id: str) -> LoggedModel:
+        """
+        Fetch the logged model with the specified ID.
+
+        Args:
+            model_id: ID of the model to fetch.
+
+        Returns:
+            The fetched model.
+        """
+        endpoint = get_logged_model_endpoint(model_id)
+        response_proto = self._call_endpoint(GetLoggedModel, endpoint=endpoint)
```

Comment:
Do we need to specify `GetLoggedModel ` is a `GET` method?

Reply:
`_call_endpoint` automatically figures out the right method :)